tests/python_on_whales/components/test_container.py

```diff
@@ -924,6 +924,102 @@ def test_prune(ctr_client: DockerClient):
     assert container not in ctr_client.container.list(all=True)
 
 
+@pytest.mark.parametrize("ctr_client", ["docker", "podman"], indirect=True)
+def test_prune_streaming(ctr_client: DockerClient):
+    for container in ctr_client.container.list(filters={"name": "test-container"}):
+        ctr_client.container.remove(container, force=True)
+    container = ctr_client.container.create("busybox")
+    assert container in ctr_client.container.list(all=True)
+
+    # container not pruned because it is not old enough
+    # podman does not provide logs when not pruned
+    logs = list(ctr_client.container.prune(filters={"until": "100h"}, stream_logs=True))
+    assert container in ctr_client.container.list(all=True)
+
+    assert len(logs) >= 1 if ctr_client.client_config.client_type == "docker" else True
+    logs_as_big_binary = b""
+    for log_type, log_value in logs:
+        assert (
+            (log_type in ("stdout", "stderr"))
+            if ctr_client.client_config.client_type == "docker"
+            else True
+        )
+
+        logs_as_big_binary += log_value
+    assert (
+        b"Total reclaimed space:" in logs_as_big_binary
+        if ctr_client.client_config.client_type == "docker"
+        else True
+    )
```

Comment:
```suggestion
    if ctr_client.client_config.client_type == "docker":
        assert b"Total reclaimed space:" in logs_as_big_binary
```
This `else True` doesn't need to be here. If writing in multiple line, there is no need to put the condition in the assert like this.